source/extensions/quic_listeners/quiche/envoy_quic_utils.h

```diff
@@ -32,9 +32,14 @@ namespace Quic {
 Network::Address::InstanceConstSharedPtr
 quicAddressToEnvoyAddressInstance(const quic::QuicSocketAddress& quic_address);
 
-quic::QuicSocketAddress envoyAddressInstanceToQuicSocketAddress(
+quic::QuicSocketAddress envoyAddressIpToQuicSocketAddress(const Network::Address::Ip* envoy_ip);
+
+quic::QuicSocketAddress envoyAddressInstancePtrToQuicSocketAddress(
     const Network::Address::InstanceConstSharedPtr& envoy_address);
 
+quic::QuicSocketAddress
+envoyAddressInstanceToQuicSocketAddress(const Network::Address::Instance& envoy_address);
+
```

Comment:
I think these 3 can be replaced with envoyAddressIpToQuicSocketAddress() with minor changes on the call sites. Can you do that?

Reply:
Converted all similar calls to `envoyAddressIpToQuicSocketAddress()`